File: 7KYU/Don't give me five!/solution.cpp
int containsFive(int n) {
  while(n != 0) {
    if(abs(n % 10) == 5) {
      return 1;
    }
    n /= 10;
  }
  return 0;
}

int dontGiveMeFive(int start, int end)
{
  int total = 0;
  for(int i = start; i <= end; i++) {
    if(containsFive(i) == 0){
      total++;
    }
  }
  return total;
}

File: 7KYU/Don't give me five!/solution.go
package kata

import (
	"math"
)

func ContainsFive(n int) bool {

	for {
		if n == 0 {
			return false
		}
		if math.Abs(float64(n%10)) == 5 {
			return true
		}
		n = n / 10
	}
	return false
}
func DontGiveMeFive(start int, end int) int {
	total := 0
	for i := start; i <= end; i++ {
		if ContainsFive(i) == false {
			total = total + 1
		}
	}
	return total
}
